source/common/common/hash.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <string>
+
+#include "xxhash.h"
+
+namespace Envoy {
+
+class HashUtil {
+public:
+  /**
+   * Return 64-bit hash with seed of 0 from the xxHash algorithm.
```

Comment:
nit: Can you link to repo/docs so people can read about it if they want.

Reply:
done